src/main/java/org/folio/ed/handler/RetrievalQueueRecordHandler.java

```diff
@@ -53,12 +53,16 @@ private String getCirculationUrl(Configuration configuration) {
   }
 
   private CaiaSoftRequest getCaiaSoftRequest(RetrievalQueueRecord retrievalRecord) {
+    log.debug("getCaiaSoftRequest:: Getting Caiasoft Request");
     var requestItem = CaiaSoftRequestItem.of(retrievalRecord.getHoldId(),
       retrievalRecord.getRequestType(), retrievalRecord.getItemBarcode(), retrievalRecord.getPickupLocation());
+    log.info("getCaiaSoftRequest:: Retrieved Caiasoft Request");
     return CaiaSoftRequest.of(Collections.singletonList(requestItem));
   }
 
   private ResponseEntity<String> post(String url, HttpHeaders headers, Object entity) {
+    log.debug("post:: Sending POST request to URL: {}", url);
+    log.info("post:: Sent POST request to URL: {}", url);
```

Comment:
Please remove info log and keep single logger statement as method itself is having single line.